_functions/synth_parameters/delay.md

```diff
@@ -2,4 +2,5 @@
 title: delay
 category: synth_parameters
 ---
-a pattern of numbers from 0 to 1. Sets the level of the delay signal.
+a pattern of numbers that set the initial level of the delay signal.  I.e. a value of one means the first echo will be as
```

Comment:
Doesn't the `SwitchDelay` get multiplied by `delayAmp`, which is negative if `delay` is negative?  Maybe careful listening is the only way to see exactly what happens, though I'm not sure if my ears can easily discern a negative feedback comb from a positive one.